decidim-forms/app/packs/src/decidim/forms/admin/forms.js

```diff
@@ -392,6 +393,8 @@ export default function createEditableForm() {
 
       autoLabelByPosition.run();
       autoButtonsByPosition.run();
+
+      initLanguageChangeSelect($field.find("select.language-change").toArray());
```

Comment:
Nice! Didn't know about `.toArray()` in jQuery, much cleaner than my suggestion!